Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "objc"
 version = "0.1.6"
 authors = ["Steven Sheldon"]
+build = "build.rs"
```

Comment:
For an example of what I mean, you can take a look at the `test_utils` crate in the block library: https://github.com/SSheldon/rust-block/tree/master/test_utils

I used a separate crate there since I only needed a build script for a test dependency and wanted to avoid adding gcc as a dependency for users.